1230-doi_minting/1233-owner_navigate_to_the_landing_page_of_a_drafted_publication_without_doi.feature

```diff
@@ -0,0 +1,7 @@
+    @1233
+    Scenario: Owner navigate to the landing page of a drafted publication without DOI
+        Given that an Owner views the landing page for their Publication
```

Comment:
hva er landing page? er det Public Publication Page?

Reply:
det er i praksis dt ja, endret etter å ha sett kommentar i saken: https://unit.atlassian.net/browse/NP-1233?atlOrigin=eyJpIjoiNjEyZTFiMjIwYzA2NDFmMWJmZjc1YzM3NTY1NWUxODkiLCJwIjoiaiJ9

Egentlig litt spørsmål om hva som er et godt begrep her også. I dette tilfellet gjelder det en draft publication, så den er jo ikke synlig for omverden enda (bare for owner), så Landing Page gir mer logisk enn Public Page i dette tilfellet, men samtidig er det forvirrende med to begreper for det samme

Reply:
jeg er med på Landing Page. Så lenge vi bruker det konsistent.

Reply:
```suggestion
        Given that an Owner views the Landing Page for their Publication
```

Reply:
filnavnet må også endres (ref skrivefeil)